src/Microsoft.Diagnostics.Monitoring.RestServer/StorageOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+#if UNITTEST
+namespace Microsoft.Diagnostics.Monitoring.UnitTests.Options
+#else
+namespace Microsoft.Diagnostics.Monitoring.RestServer
+#endif
+{
+    internal class StorageOptions
```

Comment:
Do we want to add another option to allows specifying a dump location in the "remote" container that is different in the dotnet-monitor container so that way we don't have to force a symmetry of the dump location in both containers? Maybe `public string RemoteDumpTempFolder { get; set; }`?

Reply:
Given the common usage for containers, I don't think we would need something like this.